dadi/lib/models/route.js

```diff
@@ -218,7 +218,7 @@ Route.prototype.getNetwork = function () {
 }
 
 Route.prototype.getRecipe = function () {
-  return cache.get(this._getCacheKey()).then((cachedRecipe) => {
+  return cache.getStream(this._getCacheKey()).then(cachedRecipe => {
     if (cachedRecipe) return Promise.resolve(cachedRecipe)
```

Comment:
Do you mind just changing this to:

```js
if (cachedRecipe) return cachedRecipe
```

That `Promise.resolve()` is unnecessary, as we're already inside a Promise.

Other than that, looks great!